src/routes/auth/index.ts

```diff
@@ -102,21 +102,17 @@ authRoutes.post('/sign-in', async (context: Context<EnvironmentBindings>) => {
 		throw error;
 	}
 
-	const { id: profileId, password: storedPassword } = await getProfileIdPassword(context.env.database, parsedBody.email);
-	if (!profileId || !storedPassword) {
+	const accessData = await getAccess(context.env.database, parsedBody.email);
+	if (!accessData) {
 		return context.json<StatusResponse>({ message: 'Invalid email' }, StatusCodes.UNAUTHORIZED);
 	}
+	const { id: profileId, password: storedPassword, accessLevel } = accessData;
```

Comment:
Please keep it destructured, there is no need to save to a variable to then destructure. And use as a check a field that should always be returned, like `profileId`